Singularity/Singularity/Platform/Action.cs

```diff
@@ -1,9 +1,9 @@
-﻿namespace Singularity.platform
```

Comment:
I think you can delete this file, since it was only tentative and you refactored every use of Action